### ISSUE TITLE
File Location

### ISSUE THREAD

**Issue by BenedictCodesHere** [BenedictCodesHere/prework-study-guide#11]
# File Location

## User Story

As a bootcamp student,
I want the files for my project to be in the correct directories,
so that the webpage displays correctly.
![Untitled](https://user-images.githubusercontent.com/119274509/205162859-742ef2c9-9f90-41b9-989d-90c3f0a075af.png)

##  Acceptance Criteria

GIVEN a Prework Study Guide website,
WHEN I visit the website using the URL,
THEN I can see the fully formatted version of the webpage, with relevant Javascript and CSS.

**Comment by BenedictCodesHere** [comment]
![updated](https://user-images.githubusercontent.com/119274509/205167099-23a4970d-d59f-42f6-9cda-0f583591267f.png)


Resolved. Moved all files up a directory using Git Bash

**Comment by BenedictCodesHere** [comment]
I had to add this extra Issue in the Repository and resolve it.

I had created a folder within a folder, somehow. 
The path of the directory was showing as: 

> "User 1/bootcamp/prework-study-guide/prework-study-guide"

and there were some files housed inside that path, and some just in: 

> "User1/bootcamp/prework-study-guide"

 I resolved the issue using Git Bash and by consulting the Github Docs on how to move files and push the changes to the main Repository.
